From 07cd50e678a463f23265e5baa4492671b4c20b04 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 3 Jan 2013 09:00:18 -0800
Subject: [PATCH 606/628] watchdog: xilinxps: Port driver to COMMON_CLK

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit b210be587d011688177e03b0570754dd32537cdf

Use the common clock framework to manage device clocks.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Acked-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 arch/arm/boot/dts/zynq-zc706.dts |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/zynq-zc706.dts b/arch/arm/boot/dts/zynq-zc706.dts
index d1013a3..b53b15e 100644
--- a/arch/arm/boot/dts/zynq-zc706.dts
+++ b/arch/arm/boot/dts/zynq-zc706.dts
@@ -70,7 +70,6 @@
 			device_type = "watchdog";
 			compatible = "xlnx,ps7-wdt-1.00.a";
 			reg = <0xf8005000 0x100>;
-			clock-frequency = <111111111>;
 			reset = <0>;
 			timeout = <10>;
 		};
-- 
1.7.5.4

